api-metastore/src/main/java/org/zalando/nakadi/controller/PartitionsController.java

```diff
@@ -36,19 +42,20 @@
 import org.zalando.nakadi.service.RepartitioningService;
 import org.zalando.nakadi.service.timeline.TimelineService;
 import org.zalando.nakadi.view.Cursor;
-import org.zalando.nakadi.view.CursorLag;
 import org.zalando.nakadi.view.EventTypePartitionView;
 import org.zalando.nakadi.view.PartitionCountView;
+import static org.springframework.http.ResponseEntity.noContent;
+
 
 import javax.annotation.Nullable;
-import java.util.Collections;
+import static java.util.Collections.singletonList;
+import static java.util.Collections.emptyList;
 import java.util.List;
+import java.util.Map;
 import java.util.Optional;
 import java.util.stream.Collectors;
 
-import static org.springframework.http.ResponseEntity.noContent;
-import static org.springframework.http.ResponseEntity.ok;
-
+@Validated
```

Comment:
Please confirm that `@Validated` annotation is required here.

Reply:
Yea not needed, forgot to remove it :)